### ISSUE TITLE
Atom is the Best. Shame on Microsoft the Community Stands Stronger, No Archiving Atom

### ISSUE THREAD

**Issue by iamunadike** [atom/atom#25507]
Some people may not believe , but even this person call bill gates bought github,  he and what ever noticed that atom is just too good to the developer community, Atom serves me and a lot of people alike and even in its weak state it is preferred by Many Developers and still poses a threat to the useless VScode. 
Now Ask yourself, if microsoft was not maintaining atom, why the headache to archive it, 
A/  Because the atom is making it difficult to sell useless vs code to the developers and the Bill man wants to Enforce 
Bill gates like prying into other people lives , Vaccine etc . He just looks for ways to force people like an antichrist
We WANT ATOM, NO vscode, ATOM is still my editor till date and I hope and pray the community will be stronger to Fork it and Maintain it, we dont care if we have to download it from another link.
ATOM is too good

**Comment by iamunadike** [comment]
I just feel like despite the challenges of atom, as a product, atom is exceptional do a side by side comparison of atom and vscode especiall on a unix or linux computer and see the  difference , Atom is built well and community is still appreciating because they dont paid to maintain atom, its just little by little and still even atom in its Worst state is better than Vscode, 
I really dont Wish to have to open Vscode for any reason, I already love Atom , It serves well and the community. the real deal hhere is Money, Microsoft vs.. usless... code aint selling and atom is making it harder , So you get it??

**Comment by iamunadike** [comment]
I am not gonna stop using atom, I believe the community is stronger

**Comment by gksudolol** [comment]
You sound like an entitled moron who think the world's made for them.

**Comment by gksudolol** [comment]
Cry about it moron.

**Comment by iamunadike** [comment]
you need a break, calm down boy, No one is calling you out. 
Real men not argue on github. If you need a real deal  let me know, I am only writing  my thread and how much Atom is to the community.

**Comment by gksudolol** [comment]
Touch grass snowflake.

**Comment by FHU-yezi** [comment]
> I am not gonna stop using atom, I believe the community is stronger

see #25510, they forked this project and start a discussion about the future development and community build.

**Comment by A-childs-encyclopedia** [comment]
> > I am not gonna stop using atom, I believe the community is stronger
> 
> see #25510, they forked this project and start a discussion about the future development and community build.

Tysm

**Comment by flashtechstudios** [comment]
> Someting definitely smells suspicious. Why archive atom at this point , Go on and do a survey . Atom is still appreciated by a vast majority, despite vscode is more popular,. Somehow they want to control us and besides , Vscode has many logs to Track you and whatever you do. I really dont want Microsoft near me.. esp. Vscode

Fuck 'em. Stupid idiots. They love their VSCode. Guess what?! I uninstalled that piece of shit today.

**Comment by raer0** [comment]
> Fuck 'em. Stupid idiots. They love their VSCode. Guess what?! I uninstalled that piece of shit today.

Congrats. you really showed them. I'm sure they'll bring Atom back to life because one scriptkiddie got angry and uninstalled VSCode.

**Comment by flashtechstudios** [comment]
> Cry about it moron.

stfu moron

**Comment by flashtechstudios** [comment]
> > Fuck 'em. Stupid idiots. They love their VSCode. Guess what?! I uninstalled that piece of shit today.
> 
> Congrats. you really showed them. I'm sure they'll bring Atom back to life because one scriptkiddie got angry and uninstalled VSCode.

shut up stupid idiot

**Comment by raer0** [comment]
> > > Fuck 'em. Stupid idiots. They love their VSCode. Guess what?! I uninstalled that piece of shit today.
> > 
> > 
> > Congrats. you really showed them. I'm sure they'll bring Atom back to life because one scriptkiddie got angry and uninstalled VSCode.
> 
> shut up stupid idiot

Your discourse is truly inspiring.

**Comment by flashtechstudios** [comment]
> > > > Fuck 'em. Stupid idiots. They love their VSCode. Guess what?! I uninstalled that piece of shit today.
> > > 
> > > 
> > > Congrats. you really showed them. I'm sure they'll bring Atom back to life because one scriptkiddie got angry and uninstalled VSCode.
> > 
> > 
> > shut up stupid idiot
> 
> Your discourse is truly inspiring.

Your idiotic comments are truly inspiring

**Comment by gksudolol** [comment]
> > Cry about it moron.
> 
> stfu moron

Cope you degenerate.
Type properly instead of that Internet slang.

**Comment by autumnblazey** [comment]
this issue thread is amusing

**Comment by LittleTasteOfHeaven** [comment]
@flashtechstudios use vscode and be happy with it

**Comment by flashtechstudios** [comment]
> @flashtechstudios use vscode and be happy with it

I am already happy using VSCodium : )

**Comment by FinnT730** [comment]
Come on people.... Sunsetting is NOT the end of Atom....
Sure, it will not get official support from Github/Microsoft anymore, but it has not gotten that over the past few years.

**Have you really forgotten that this is on Github, and you can just fork it, and keep using it? Patch all the broken things out, or hell, even create a new editor out of it? _It is NOT DEAD after the sunsetting, it has been dead for the past few years, and people still use it._ Windows XP has been dead for years, and people use it every day still......**

**Comment by Binomial-theorem** [comment]
facts

**Comment by flashtechstudios** [comment]
> Come on people.... Sunsetting is NOT the end of Atom.... Sure, it will not get official support from Github/Microsoft anymore, but it has not gotten that over the past few years.
> 
> **Have you really forgotten that this is on Github, and you can just fork it, and keep using it? Patch all the broken things out, or hell, even create a new editor out of it? _It is NOT DEAD after the sunsetting, it has been dead for the past few years, and people still use it._ Windows XP has been dead for years, and people use it every day still......**

true

**Comment by softorangetech11** [comment]
Checkmark if it is better to do that option...

- [ ] Just use VS Code
- [ ] Use Atom after Dec. 15, 2022
- [ ] Fork Atom and make your own
- [x] Use Atom-community builds

**Comment by beritasewu** [comment]
i dont believe it [Berita Terupdate Hari ini](https://beritasewu.com/) anymore.

**Comment by Xmetalfanx** [comment]
> Come on people.... Sunsetting is NOT the end of Atom.... Sure, it will not get official support from Github/Microsoft anymore, but it has not gotten that over the past few years.

EXACTLY ... i use to love Atom but it started to seem clunky to me and not getting TLC it needs ... found out that vscode is so similar (i dont get people acting like they are like totally different animals UX/UI wise) and that it fit me better .... this may be exactly what Atom needs and I'd love to give a fork a try (using it) ... one of the first things I would love to see in Atom's fork is for it to be faster overall and to maybe address long standing "issues" that have been issues for years ... This seems like it could be a great thing ... the community seems to love Atom so they (Linux user/Open Source fan here) can keep it going and give it the love it needs

**Comment by ItsAleph** [comment]
Atom is best. dot.
